src/main/java/com/amazon/opendistroforelasticsearch/jdbc/config/FetchSizeProperty.java

```diff
@@ -0,0 +1,14 @@
+package com.amazon.opendistroforelasticsearch.jdbc.config;
+
+public class FetchSizeProperty extends IntConnectionProperty{
+
+    public static final String KEY = "fetchSize";
+
+    public FetchSizeProperty() {
+        super(KEY);
+    }
+
+    public Integer getDefault() {
+        return 0;
```

Comment:
Should default value be `null` or 0? Any difference?

Reply:
The default behavior should be non-paginated query. It is supported by SQL plugin by making `fetch_size=0`. So default is set to `0`.

Reply:
Do we need override the getDefault already been implemented in IntConnectionProperty?

Reply:
Yes, not necessarily needed here. I felt it's easy for the reader to know that default value is zero in `FetchSizeProperty` instead of checking in `IntConnectionProperty`. Will remove for sake of brevity.